src/Resources/config/validator.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumberValidator">
+            <tag name="validator.constraint_validator" />
+            <argument type="service" id="libphonenumber\PhoneNumberUtil" />
```

Comment:
we should probably explicitely declare this service. People can disable auto discovery and all.

Reply:
It is declared in [services.xml](https://github.com/odolbeau/phone-number-bundle/blob/master/src/Resources/config/services.xml#L9)